Source/WebCore/platform/audio/gstreamer/AudioFileReaderGStreamer.cpp

```diff
@@ -389,7 +389,7 @@ void AudioFileReader::plugDeinterleave(GstPad* pad)
 
 void AudioFileReader::decodeAudioForBusCreation()
 {
-    ASSERT(&m_runLoop == &RunLoop::current());
+    ASSERT(m_runLoop.isCurrent());
```

Comment:
i think now it's better to use:
   assertIsCurrent(m_runLoop);
So you get the thread safety analysis should the future variable declarations start using it.